src/manifest.json

```diff
@@ -44,6 +44,16 @@
   "content_scripts": [
     {
       "js": [
+        "js/first_parties.js"
+      ],
+      "matches": [
+        "<all_urls>"
```

Comment:
I suggest injecting the Twitter-specific script directly, and only for Twitter URLs (international too). This way we avoid making any changes whatsoever to any non-Twitter pages.